src/main/java/com/buddy/api/domains/account/services/CreateAccountService.java

```diff
@@ -0,0 +1,7 @@
+package com.buddy.api.domains.account.services;
+
+import com.buddy.api.domains.account.dtos.AccountDto;
+
+public interface CreateAccountService {
```

Comment:
```suggestion
public interface CreateAccount {
```
Para seguir o padrão de nomenclatura dos outros services.